packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -866,14 +880,75 @@ export default function makeKernelKeeper(kernelStorage, kernelSlog) {
     kvStore.set(`${kernelSlot}.data.slots`, capdata.slots.join(','));
   }
 
-  function cleanupAfterTerminatedVat(vatID) {
+  function removeVat(vatID) {
```

Comment:
A name like "removeVat" suggests effects stronger than mere omission from exports.
```suggestion
  function stopStoreExport(vatID) {
```

Reply:
I'll use `removeVatFromSwingStoreExports`, which also lets me remove the comment line